test/e2e/specs/preview.test.js

```diff
@@ -97,16 +97,6 @@ describe( 'Preview', () => {
 		// Preview for published post (no unsaved changes) directs to canonical URL for post.
 		await editorPage.bringToFront();
 		await publishPost();
-		// Wait until the publish panel is closed
-		await Promise.all( [
-			editorPage.waitForFunction( () => ! document.querySelector( '.editor-post-publish-panel' ) ),
-			editorPage.click( '.editor-post-publish-panel__header button' ),
-		] );
-		expectedPreviewURL = await editorPage.$eval( '.components-notice.is-success a', ( node ) => node.href );
-
-		await editorPage.bringToFront();
-		await waitForPreviewNavigation( previewPage );
-		expect( previewPage.url() ).toBe( expectedPreviewURL );
```

Comment:
I removed this part because in my testing even if the url is a preview url and not the final url, it works as intended.

Reply:
It works this way in classic, so maybe we should keep it with this param.

Reply:
I don't remember why this part is here, @aduth should confirm whether we can remove it. I remember that this test was very fragile in the past.

Reply:
I think the idea was that if there is nothing to save and we have a known preview URL, we don't save, we go straight to the preview URL.

I don't know that it's necessarily breaking to let the save go through, however.